packages/main/src/Select.js

```diff
@@ -424,6 +428,29 @@ class Select extends UI5Element {
 		}
 	}
 
+	valueStateTextMappings() {
```

Comment:
This should logically be a getter, please change it

Reply:
Done